server/lib/routes/get-client.json.js

```diff
@@ -24,7 +24,8 @@ module.exports = function (i18n) {
     res.set('Vary', 'accept-language');
 
     // charset must be set on json responses.
-    res.charset = 'utf-8';
+    // set charset for Express 4: https://github.com/strongloop/express/wiki/Migrating-from-3.x-to-4.x#rescharset
+    res.set('content-type', 'application/json; charset=utf-8');
```

Comment:
@vladikoff - Does this change also need to be made in [get-ver-json.js](https://github.com/mozilla/fxa-content-server/blob/b6e5b00fa4598787abae10eaeb6f74b8c816684b/server/lib/routes/get-ver.json.js#L126)?